  * debian/patches/fix_symbols_hr.diff: New patch to fix Croatian layout,
    some differences with previous XFree86 keymap had been introduced
    in xkeyboard-config.  Closes: #379292  Thanks Josip Rodin

Submitted upstream: https://bugs.freedesktop.org/show_bug.cgi?id=8019

Index: symbols/hr
===================================================================
--- symbols/hr.orig
+++ symbols/hr
@@ -5,6 +5,12 @@
     name[Group1]="Croatia";
 
     include "cs(latin)"
+    // Redefine these keys to match XFree86 Croatian layout
+    key <AE01> { [         1,     exclam,   asciitilde,   dead_tilde ] };
+    key <AE03> { [         3, numbersign,  asciicircum, dead_circumflex ] };
+    key <AE05> { [         5,    percent,       degree, dead_abovering ] };
+    key <AE07> { [         7,      slash,        grave,   dead_grave ] };
+    key <AB10> { [     minus, underscore, dead_belowdot, dead_abovedot ] };
 };
 
 partial alphanumeric_keys 
